validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -25,4 +25,17 @@ class LarQualityEngineSpec
       }
     }
   }
+
+  property("There should be no duplicates in qualityEditsList") {
+    qualityEditsList.distinct.length mustBe qualityEditsList.length
+  }
+
+  property("There should be edits equal to the number of edit files - 1 (for Q022)") {
```

Comment:
This is a pretty brittle way of determining the number of edits that _should_ be present.  What if some utility class file is added to that directory?  What if we want to turn off an edit by commenting it out of the list?  We'd have to delete the corresponding file as well.  It also doesn't test that its the expected set of edits, just the correct number.